### DISCUSSION TITLE
Where to find document

### DISCUSSION THREAD

**Discussion by aplqo** [tweag/nickel#483] [Q&A] [answered]
Is there any document about syntax, and where is it?

**Comment by yannham** [comment]
Hi, unfortunately there isn't currently anything yet about syntax. You can look at the examples to get a sense of it. I'm writing currently the basic documentation on typing, syntax, contracts and so on to have them for the first release, but it's WIP. In the meantime, don't hesitate to ask more specific questions if you have any issue with a specific syntax point

**Comment by yannham** [comment, marked as answer]
Update: the syntax section is now part of the manual. See https://nickel-lang.org/user-manual/syntax.
